Chapter_2_Getting_Started.md

```diff
@@ -0,0 +1,1206 @@
+
+When executing Robot Framework, it either parses directory trees or files, depending on which paths are given.
+
+The given path to Robot Framework where it starts parsing is considered the **Root Suite**.
+
+If the path to a single file is given as **Root Suite** directly to Robot Framework, only this file is parsed.
+
+If a directory path is given, starting at this location, Robot Framework will parse all `*.robot` files and directories within this path.
+Robot Framework analyzes all containing files and determines if they contain test cases or tasks. If they do, they are considered **Suite Files** or **Low-Level Suites**.
+All directories that either directly or indirectly contain a Suite File are considered **Suites Directories** or **Higher-Level Suites**.
+
+The ordering of suites during execution is, by default, defined by their name and hierarchy.
+All files and directories, which are suites in one directory, are considered on the same level and are executed in case-insensitive alphabetical order.
+
+
+It is possible to define the order without influencing the name of the suite by adding a prefix followed by two underscores `__` to the name of the directory or file. This prefix is NOT considered part of the name.
+So `01__First_Suite.robot` sets the suite name to `First Suite`, while `2_Second_Suite.robot` sets the suite name to `2 Second Suite`.
+
+One or more underscores in file or directory names are replaced by space characters as suite names.
+
+Legend:
+```plaintext
+▷ Directory (No Suite)
+▶︎ Suite Directory
+◻︎ File (No Suite)
+◼︎ Suite File
+```
+
+Example:
+```plaintext
+ ----- Tree Structure / Order --------- | ---- Suite Name ---------
+  ▶︎ Root_Suite                          | Root Suite
+    ◼︎ A_Suite.robot                     | A Suite
+    ▶︎ Earlier_Suite_Directory           | Earlier Suite Directory
+      ◼︎ B_Suite.robot                   | B Suite
+      ◼︎ C_Suite.robot                   | C Suite
+    ▷ Keywords (No Suite)               |
+      ◻︎ technical_keywords.resource     |
+    ▶︎ Later_Suite_Directory             | Later Suite Directory
+      ◼︎ 01__FirstSuite.robot            | First Suite
+      ◼︎ 02__SecondSuite.robot           | Second Suite
+      ▶︎ 03__ThirdSuite                  | Third Suite
+        ◼︎ 01__FirstSubSuite.robot       | First Sub Suite
+        ◼︎ 02__SecondSubSuite.robot      | Second Sub Suite
+      ◼︎ 04__FourthSuite.robot           | Fourth Suite
+```
+
+
+
+### 2.1.1 Suite Files
+
+> [!IMPORTANT]
+> LO-2.1.1 Recall the conditions and requirements for a file to be considered a Suite file (K1)
+
+Robot Framework parses files with the extension `.robot` and searches for test cases or tasks within these files.
+
+A parsed file that contains at least one test case or task is called a **Suite File**.
+
+A Suite File **either** contains `*** Test Cases ***` (in Test Suites) **or** `*** Tasks ***` (in Task Suites), but it CANNOT contain both types simultaneously.
+
```

Comment:
It should probably be mentioned that every suite file under the same root suite should only contain either tests case or tasks.

Reply:
@datakurre That is a recommendation, but i wouldn't like loading the syllabus with more recommendations.
I will probably remove some of the recommendation that we already have there, because we will maybe not ask for them in the exam.

Reply:
But doesn't `robot` give you error and require to choose either `--rpa` or `--norpa` if the parsed suite as whole contains both?